docs/General/buildRPC.md

```diff
@@ -31,7 +31,7 @@ Read more about GitHub Actions [here](https://github.blog/2019-08-08-github-acti
 
 3. Click the **Run Workflow** dropdown. 
 
-4. Select the **branch: v1.2.2** from the **Use workflow from** dropdown. 
+4. Select the **branch: v1.2.2** from the **Use workflow from** dropdown. If you don't see it in the list, type "v1.2.2" to find the correct branch.
```

Comment:
Currently we are at 1.2.1.  I'm wondering if we should rephrase this to avoid using specific version numbers and instead use latest release version as the branch value?

Reply:
Good idea.